TACC/Core-CMS-Custom
# Data Refresh and Display Issues

## Overview

Fixes multiple small issues with extensions listing including submit not refreshing data display properly and small styling issues. 

## Related

[WP-739](https://tacc-main.atlassian.net/browse/WP-739)

## Changes

- edit modal should properly display data and new data should be refetched on submit
- view modal should display data properly
- both modals should have styling more consistent with production

## Testing

1. go to http://localhost:8000/administration/list-extensions/
2. open edit modal and verify record has an Applicable Data Period (a few records do not, this is simply bad data, please disregard and select another that has this field populated)
3. Edit Applicable Data Period and verify it successfully submits, and the Applicable Data Period display field under the form both updates and displays properly
4. Edit any other field and verify the same
5. Close the modal and make sure the record was properly update in the listing
6. Verify both Edit and View modals seem consistent with other modals
7. Verify other extensions are displaying data properly even after first record has been viewed/edited

## UI

…
